src/config/server.rs

```diff
@@ -99,6 +99,11 @@ pub struct Server {
     /// `workers` sets the number of worker threads. @default the number of
     /// system cores.
     pub workers: Option<usize>,
+
+    #[serde(default, skip_serializing_if = "is_default")]
+    /// `dedupe_io` if true, will use caching to prevent
+    /// duplicate IO calls. @default `false`.
```

Comment:
```suggestion
    /// When set to `true`, it will ensure no HTTP, GRPC, or any other IO call is made more than once within the context of a single GraphQL request.
```

Reply:
updated the doc comment